packages/api/src/entities/PaymentFlatEntity.ts

```diff
@@ -54,46 +57,118 @@ export default class PaymentFlatEntity {
+            adapter: value => {
+                if (!value) return value;
+                return parseInt(value, 10);
+            },
+        },
+        typeIdEtablissementBeneficiaire: {
+            path: ["Code taxe 1"],
+            adapter: value => {
+                if (Siret.isSiret(value)) return "siret";
+            },
+        },
+        idEtablissementBeneficiaire: {
+            path: ["Code taxe 1"],
+            adapter: value => {
+                if (value) return new Siret(value);
+            },
+        },
+        typeIdEntrepriseBeneficiaire: {
+            path: ["Code taxe 1"],
+            adapter: value => {
+                if (Siret.isSiret(value)) return "siren";
+            },
+        },
+        idEntrepriseBeneficiaire: {
+            path: ["Code taxe 1"],
+            adapter: value => {
+                if (value) return new Siret(value).toSiren();
+            },
+        },
+        amount: {
+            path: ["Montant payé"],
+            adapter: value => {
+                if (!value || typeof value === "number") return value;
+
+                return parseFloat(value.replace("\r", "").replace(" ", "").replace(",", "."));
+            },
+        },
+        dateOperation: {
+            path: ["Date de dernière opération sur la DP"],
+            adapter: value => {
+                if (!value) return value;
+                if (value != parseInt(value, 10).toString()) {
+                    const [day, month, year] = value.split(/[/.]/).map(v => parseInt(v, 10));
+                    return new Date(Date.UTC(year, month - 1, day));
+                }
+                return GenericParser.ExcelDateToJSDate(parseInt(value, 10));
+            },
+        },
+
+        centreFinancierCode: {
+            path: ["Centre financier CODE"],
+        },
+        centreFinancierLibelle: {
+            path: ["Centre financier"],
+        },
+        attachementComptable: {
+            path: ["Société"],
+        },
+        ej: {
+            path: ["N° EJ"],
+        },
+        provider: {
+            path: [],
+            adapter: () => "Chorus",
+        },
+    };
+
     public regionAttachementComptable: string | null;
+    public idVersement: string;
+    public uniqueId: string;
 
     constructor(
-        public uniqueId: string,
-        public idVersement: string,
         public exerciceBudgetaire: number,
         public typeIdEtablissementBeneficiaire: typeIdEtablissement,
         public idEtablissementBeneficiaire: idEtablissementType<typeIdEtablissement>,
         public typeIdEntrepriseBeneficiaire: typeIdEntreprise,
         public idEntrepriseBeneficiaire: idEntrepriseType<typeIdEntreprise>,
         public amount: number,
         public operationDate: Date,
+        public centreFinancierCode: string,
+        public centreFinancierLibelle: string,
+        public attachementComptable: string,
+        public ej: string,
+        public provider: string,
```

Comment:
ça ne peut plus être nul..? On sait le remplir pour fonjep ?

Reply:
Ah oui je vois que ça ne peut pas être nul car on l'utilise pour construire les ids. Mais je maintient la question : sait-on le remplir dans le cas général ?

Reply:
J'ai donné à centreFinancierLibelle la possibilité d'être null. Pour les autres, on ne sais pas remplir dans le cas generale mais dans ce cas je mettrais plutôt une string du type "N/A" ou "not available for this provider" pour que on sache que que ce n'est pas une valeur manquante car mauvaise qualité des données mais plutôt une valeur manquante car pas pertinent pour ce provider. Qu'est-ce que t'en penses ?

Reply:
ok mais à documenter alors. On peut mettre comme type `string |"N/A"` par exemple. Ce sera pas très significatif en typescript mais ça clarifie